homeassistant/components/geniushub/sensor.py

```diff
@@ -0,0 +1,88 @@
+"""Support for Genius Hub sensor devices."""
+import logging
+from time import (localtime, strftime)
+
+from homeassistant.const import (DEVICE_CLASS_TEMPERATURE, TEMP_CELSIUS)
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+GH_HAS_BATTERY = [
+    'Room Thermostat', 'Genius Valve', 'Room Sensor', 'Radiator Valve']
+
+GH_STATE_ATTRS = [
+    'setTemperature', 'luminance', 'occupancyTrigger', 'batteryLevel']
```

Comment:
These should instead be individual sensors, and not state attributes.

Reply:
`'occupancyTrigger'` sounds like a binary sensor. What kind of measurement is it?

Reply:
TBH, I truly dunno - I was including it only because the underlying RESTful API reports it. 

I will take it out for now.  I only require `lastComms`, `location` and `batteryLevel` (see above).